src/modules/user/components/LogOut.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { FirebaseService } from 'modules/firebase';
+
+const LogOut = () => {
+  const firebase = FirebaseService.Instance;
+  const auth = firebase.auth();
+
+  const handleLogout = React.useCallback(async () => {
+    await auth.signOut();
```

Comment:
async/await here is actually not required.

We are just holding this code back, but not doing anything with result, as it fires as a callback inside onAuthStateChange method from firebase sdk.